custom_conf.py

```diff
@@ -59,6 +59,29 @@
 # (default is the circle of friends)
 html_favicon = '.sphinx/_static/favicon.png'
 
+
+#############################################################
+# Display the contributor
+import requests
+
+def get_contributors(username, repository):
+    url = f"https://api.github.com/repos/{username}/{repository}/contributors"
+    response = requests.get(url)
+    try:
+        response.raise_for_status()
+        contributors = response.json()
+        contributors_list = [{'name': contributor['login'], 'url': contributor['html_url']} for contributor in contributors]
+        return contributors_list
+    except requests.exceptions.HTTPError as err:
+        print(f"Failed to fetch contributors: {err}")
+        return None
+
+username = 'canonical'
+repository = 'sphinx-docs-starter-pack'
```

Comment:
If you check the definition of `html_context` below, you see there is already a `github_url` field that users specify. This field contains both the organization and the repository.

We don't want users to have to specify the same information twice, so we should reuse this information here. We can either parse the `github_url` field to extract the two different parts, or we can have them specify the two different variables and then compose both the `github_url` field and the API url.